hack/chains/README.md

```diff
@@ -28,6 +28,11 @@ Wait a while until you see mostly healthy/synced at [Argo CD](https://openshift-
 matters for the basic pipeline functionality needed to do builds and
 demonstrate chains.)
 
+### (Optional) Deploy local cluster rekor server
+
+    cd hack/chains
+    ./deploy-local-rekor.sh
+
```

Comment:
It would be nice to mention how to configure chains to use that rekor instance here. I guess it's this:

    ./config rekor-local

Reply:
..or maybe the deploy script could just assume you want that and do that for you.